package.json

```diff
@@ -23,6 +23,7 @@
     "vue-datocms": "^4.0.5"
   },
   "devDependencies": {
+    "@datocms/cli": "^1.1.7",
```

Comment:
Could also add a datocms.config.js? I've branched from your branch for my feature, but when creating a new migration, I'm getting an error:
```
 ›   Error: No config file found in "datocms.config.json"
 ›   Try this: Configure a local configuration profile with "datocms profile:set"
```